### PR TITLE
Rename output to report

### PR DESCRIPTION
**Pull request by rebkwok**
This should rename output to report everywhere except for the app name itself (which is trickier, without nuking the database - which we can totally do, if we want to)

### CODE REVIEW

File: outputs/migrations/0011_datamigration_populate_report_html_field.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("outputs", "0010_report_report_html_file_path"),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            populate_report_html_file_path, populate_output_html_file_path
```
Comment (ghickman) [inline outputs/migrations/0011_datamigration_populate_report_html_field.py:30]: Why not a RenameField operation instead of the copying flow here?

**Review by ghickman — APPROVED** [review]
Looks good!  One query about the way the migrations were done, but it's not blocking.

I think renaming the app now makes sense, it's only going to get harder down the line when we have production data in the db, so we might as well do it now and hope for no more renames 🤞

File: outputs/models.py [hunk 2]
```diff
@@ -80,7 +80,7 @@ class Report(models.Model):
     description = models.TextField(
         null=True,
         blank=True,
-        help_text="Optional description to display before rendered output",
+        help_text="Optional description to display before rendered report   ",
```
Comment (benbc) [inline outputs/models.py:83]: Extra spaces snuk in here.

**Review by benbc — APPROVED** [review]
That was quick. :-)

File: outputs/migrations/0011_datamigration_populate_report_html_field.py (hunk 1 above)
Comment (rebkwok) [reply outputs/migrations/0011_datamigration_populate_report_html_field.py:30]: When I just renamed it, it wanted to add a default for the new field, which at the time I thought was going to replace the old existing filenames, but I guess it wouldn't if they already had values

**Review by ghickman — APPROVED** [review]

**Pull request merged**